master/src/main/java/org/apache/celeborn/service/deploy/master/clustermeta/AbstractMetaManager.java

```diff
@@ -111,6 +112,23 @@ public void updateAppLostMeta(String appId) {
     appHeartbeatTime.remove(appId);
   }
 
+  public void updateWorkerExcludeMeta(
+      String host,
+      int rpcPort,
+      int pushPort,
+      int fetchPort,
+      int replicatePort,
+      String excludeAction) {
+    WorkerInfo worker = new WorkerInfo(host, rpcPort, pushPort, fetchPort, replicatePort);
+    if ("ADD".equalsIgnoreCase(excludeAction)) {
+      excludedWorkers.add(worker);
```

Comment:
If excludedWorkers merged with manuallyExcludedWorkers, then excludedWorkers should contains all manuallyExcludedWorkers at any time.

Reply:
@RexXiong, I don't understand when  `excludedWorkers` does not contain all `manuallyExcludedWorkers`.